public/src/components/ToastManager.jsx

```diff
@@ -9,10 +11,10 @@ import {
 
 const getCSSForFlavor = (flavor) => {
   const flavors = {
-    success: `
+    success: css`
       background-color: #386C0B;
```

Comment:
Consider what you see the flavors changing to. If they're always just a different background color then this could just be the hex value (preferably theme property name) and the code simplified

Reply:
Fixed!